src/app/lib/utilities/preprocessor/rules/__snapshots__/topstories.test.js.snap

```diff
@@ -1461,7 +1461,7 @@ Object {
           },
         ],
         "strapline": Object {
-          "name": "Isi akụkọ",
+          "name": "Top Stories",
```

Comment:
Why is this going from Igbo to English?

Reply:
Its because of https://github.com/bbc/simorgh/pull/4661#discussion_r346226415. And falling test should be ignored first because I wanted to a way to test the `top-stories` in all services display correctly.